src/Modules/ContentBlocks/Domain/ContentBlock/ContentBlock.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace ForkCMS\Modules\ContentBlocks\Domain\ContentBlock;
+
+use Doctrine\ORM\Mapping as ORM;
+use ForkCMS\Modules\Backend\Domain\Action\ModuleAction;
+use ForkCMS\Modules\Internationalisation\Domain\Locale\Locale;
+use DateTime;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGrid;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridActionColumn;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridMethodColumn;
+use Pageon\DoctrineDataGridBundle\Attribute\DataGridPropertyColumn;
+
+#[ORM\Entity(repositoryClass: ContentBlockRepository::class)]
+#[ORM\Table(name: 'contentblocks__contentblock')]
+#[DataGrid('ContentBlock')]
+#[ORM\HasLifecycleCallbacks]
+#[DataGridActionColumn(
+    route: 'backend_action',
+    routeAttributes: [
+        'module' => 'content_blocks',
+        'action' => 'content_block_edit',
+    ],
+    routeAttributesCallback: [self::class, 'dataGridEditLinkCallback'],
+    label: 'lbl.Edit',
+    class: 'btn btn-primary btn-sm',
+    iconClass: 'fa fa-edit',
+    requiredRole: ModuleAction::ROLE_PREFIX . 'CONTENT_BLOCKS__CONTENT_BLOCK_EDIT',
+    columnAttributes: ['class' => 'fork-data-grid-action'],
+)]
+final class ContentBlock
```

Comment:
Class is not allowed to be final because it is an entity

Reply:
already changed